.env

```diff
@@ -0,0 +1 @@
+NEWRELIC_LICENSE_KEY=
```

Comment:
Should this be .env.example like in safecastapi?

Reply:
Sorry for late response. Yes, it should, so I renamed it.